File: HealthSysHub.Web.Utility/Models/HospitalDepartmentInformation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HealthSysHub.Web.Utility.Models
{
    public class HospitalDepartmentInformation
    {
        public Guid? HospitalDepartmentId { get; set; }
        public Guid? HospitalId { get; set; }
        public Guid? DepartmentId { get; set; }
        public string? DepartmentName { get; set; }
        public string? HeadOfDepartment { get; set; }
        public Guid? CreatedBy { get; set; }
        public DateTimeOffset CreatedOn { get; set; }
        public Guid? ModifiedBy { get; set; }
        public DateTimeOffset? ModifiedOn { get; set; }
        public bool IsActive { get; set; }
    }
}

File: HealthSysHub.Web.Pharmacy.UI/src/app/models/hospital-department-information.ts
export interface HospitalDepartmentInformation {
  hospitalDepartmentId?: string | null;
  hospitalId?: string | null;
  departmentId?: string | null;
  departmentName?: string | null;
  headOfDepartment?: string | null;
  createdBy?: string | null;
  createdOn: Date | string;
  modifiedBy?: string | null;
  modifiedOn?: Date | string | null;
  isActive: boolean;
}
